packages/story-editor/src/components/library/panes/shared/libraryMoveable.js

```diff
@@ -146,7 +149,13 @@ function LibraryMoveable({
     [isDragging, resetMoveable]
   );
 
+  const { clearTransforms, pushTransform } = useTransform((state) => ({
```

Comment:
👍 for clearing the transforms, too.

`clearTransforms` is used as a dep above already, probably should move this selector to before `resetMoveable`.